jujugui/static/gui/src/app/app.js

```diff
@@ -1757,6 +1762,14 @@ YUI.add('juju-gui', function(Y) {
       console.log('switching to new socket URL:', socketUrl);
```

Comment:
These are just developer messages. We have discussed moving them to a custom logging solution but right now they are quite helpful for debugging.